### DISCUSSION TITLE
I have a new collaborator on a project who wants me grant administration permissions on github--how do we do this?

### DISCUSSION THREAD

**Discussion by t12679** [community/community#61917] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I have a new collaborator on a project who wants me grant administration permissions on github--how do we do this?

**Comment by nirav-gajera** [comment]
1. Go to the repository that you want to grant administration permissions to.
2. Click on the **Settings** tab.
3. In the **Access** section, click on **Collaborators & teams**.
4. Under **Manage access**, click on the **+ Add collaborator** button.
5. Enter the collaborator's GitHub username or email address.
6. Select the **Admin** role from the **Role** dropdown menu.
7. Click on the **Add collaborator** button.


* You can revoke administration permissions at any time by following the same steps above and selecting the **Remove collaborator** button instead of the **Add collaborator** button.

**Comment by t12679** [comment]
Role dropdown does not appear.
Says he is already having access:
[image: image.png]

**Reply by nirav-gajera** [reply]
collaborator already has write access to the repository, which is why the role dropdown is not appearing. To grant them administration permissions, you will need to revoke their write access and then re-invite them with the admin role.

To do this, follow these steps:

1. Go to the repository that you want to grant administration permissions to.
2. Click on the **Settings** tab.
3. In the **Access** section, click on **Collaborators & teams**.
4. Under **Manage access**, find the collaborator's name and click on the **Remove collaborator** button.
5. Click on the **Remove collaborator** button again to confirm.
6. Invite the collaborator again with the admin role.

**Comment by t12679** [comment]
I did all that, but a drop down menu never appeared.

**Comment by t12679** [comment]
So we succeeded in adding him as collaborator, but now I am I getting this
error:
[image: image.png]
So I changed this to CNAME and pointed to t12679.github.io on hte right
most text field on google domains and it went through. But when I hit
"check again" I still go this same error message.
